tests/routes/pro/post_offerer_api_key_test.py

```diff
@@ -33,6 +33,11 @@ def test_generate_api_key(client):
     )
     assert response.status_code == 200
 
+    # test user can delete the generated api key
+    response = TestClient(client).with_auth(user_offerer.user.email).delete(f"/offerers/api_keys/{saved_key.prefix}")
+    assert response.status_code == 204
+    assert not ApiKey.query.filter_by(prefix=saved_key.prefix).first()
```

Comment:
```suggestion
    assert not ApiKey.query.count() == 0
```

... comme ça, on ne risque pas d'avoir un test qui passe à tort si jamais on se trompe dans le `filter_by()`. :)